query/src/sql/planner/binder/bind_context.rs

```diff
@@ -44,6 +44,11 @@ pub struct ColumnBinding {
     /// Another example is aggregation. In a `GROUP BY` context, aggregate funtions
     /// will be extracted and be added to `BindContext` as a `ColumnBinding`.
     pub scalar: Option<Box<Scalar>>,
+
+    /// Consider the sql: `select * from t join t1 using(a)`.
+    /// The result should only contain one `a` column.
+    /// So we need mark `t.a` or `t1.a` duplicated.
+    pub duplicated: bool,
```

Comment:
`duplicated` is a bit confusing. Maybe it's rename to `visibleInUnqualifiedWildcard`.